features/rt/hack/generate.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
hm can we move it to the root/hack directory I think everyone can use it?

Reply:
I'd suggest to merge this and #26
and the see what we can consolidate
makes sense?
/cc @karmab